package.json

```diff
@@ -34,10 +34,11 @@
     "grunt-contrib-uglify": "~0.5.0",
     "jasmine": "^2.5.1",
     "jsdom": "^9.5.0",
-    "jsonfile": "^2.3.1"
+    "jsonfile": "^2.3.1",
+    "underscore": "^1.8.3"
   },
   "scripts": {
-    "tsc": "tsc src/json-patch-duplex.ts --noEmitHelpers && tsc src/json-patch.ts --noEmitHelpers",
+    "tsc": "tsc -w",
```

Comment:
I would leave `npm run tsc` just to compile, for watch, we can create another task